jdaviz/models/physical_models.py

```diff
@@ -17,6 +17,9 @@
 
 __all__ = ["BlackBody"]
 
+# ASTROPY_LT_5_3
+__doctest_requires__ = {"BlackBody": ["astropy<5.3"]}
```

Comment:
Might need this. Not sure yet.
```suggestion
__doctest_requires__ = {"BlackBody": ["astropy<5.3,dev"]}
```

Reply:
No, we don't need dev tag.